service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/WorkspaceCloneUtils.java

```diff
@@ -164,6 +165,28 @@ public static ControlledGcsBucketResource buildDestinationControlledGcsBucket(
         .build();
   }
 
+  public static ControlledAzureStorageContainerResource buildDestinationControlledAzureContainer(
+      ControlledAzureStorageContainerResource sourceContainer,
+      UUID storageAccountId,
+      UUID destinationWorkspaceId,
+      UUID destinationResourceId,
+      String name,
+      @Nullable String description,
+      String cloudInstanceName) {
```

Comment:
Why `cloudInstanceName` for the storage container name?

Reply:
It would be good to have javadoc on this and explain the params.

Reply:
See update doc [here](https://github.com/DataBiosphere/terra-workspace-manager/blob/1b725dff35e3e1667f36eb1e7aad95f19269907f/service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/WorkspaceCloneUtils.java#L168-L180)